Godeps/Godeps.json

```diff
@@ -406,6 +406,11 @@
 			"Rev": "a1b8ba5163b7f041b22761461eabd02b70d1f824"
 		},
 		{
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Comment": "v0.1-125-g82d16f7",
```

Comment:
This is also updated by:
https://github.com/coreos/rkt/pull/2691, see:
```
+		{
 +			"ImportPath": "github.com/gogo/protobuf/proto",
 +			"Comment": "v0.2",
 +			"Rev": "4168943e65a2802828518e95310aeeed6d84c4e5"
 +		},
```
Which one should we take first? :)

It should also use a tagged version.